cromwell_tools/cromwell_auth.py

```diff
@@ -138,7 +138,7 @@ def from_user_password(cls, username, password, url):
 
     @classmethod
     def harmonize_credentials(
-            cls, username=None, password=None, url=None, secrets_file=None, caas_key=None):
+            cls, username=None, password=None, url=None, secrets_file=None, service_account_key=None):
```

Comment:
You'll need to rename the `caas_key` parameter wherever it is used in the CLI as well: https://github.com/broadinstitute/cromwell-tools/blob/master/cromwell_tools/cli.py#L94

Reply:
Good catch! will do!